wagtail/admin/tests/pages/test_reorder_page.py

```diff
@@ -0,0 +1,112 @@
+from unittest import mock
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group, Permission
+from django.contrib.messages import constants as message_constants
+from django.http import HttpRequest, HttpResponse
+from django.test import TestCase
+from django.urls import reverse
+
+from wagtail.admin.navigation import get_explorable_root_page
+from wagtail.core.models import Page
+from wagtail.core.signals import post_page_move, pre_page_move
+from wagtail.tests.testapp.models import BusinessChild, BusinessIndex, SimplePage
```

Comment:
`SimplePage` isn't used anymore in this testcase.

We should also have a test for this as ealier (not only parent/child subpages types).

I do see that when you have SimplePage with SimplePage's as childs (same page type) this actually DOES WORK currently, so we should cover that as well in the test case.